### PR TITLE
Recharts Graphing Docs

### PR DESCRIPTION
**Pull request by Alek99**

### CODE REVIEW

**Comment by masenf** [comment]
I can't seem to run it...

```console
  File "/Users/masenf/code/reflex-dev/reflex/reflex/reflex.py", line 146, in run
    prerequisites.get_app()
  File "/Users/masenf/code/reflex-dev/reflex/reflex/utils/prerequisites.py", line 114, in get_app
    app = __import__(module, fromlist=(constants.APP_VAR,))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/masenf/code/reflex-dev/reflex-web/pcweb/pcweb.py", line 6, in <module>
    from pcweb.component_list import component_list
  File "/Users/masenf/code/reflex-dev/reflex-web/pcweb/component_list.py", line 191, in <module>
    [rx.GraphingTooltip],
     ^^^^^^^^^^^^^^^^^^
AttributeError: module 'reflex' has no attribute 'GraphingTooltip'. Did you mean: 'graph_tooltip'?
```

File: docs/library/graphing/brush.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                    data_key="uv",
+                    stroke="#8884d8",
+                    fill="#8884d8"
+                ), 
+                rx.bar(
+                    data_key="pv",
+                    stroke="#82ca9d",
+                    fill="#82ca9d" 
+                ), 
+                rx.brush(data_key="name", height=30, stroke="#8884d8"),
```
Comment (masenf) [inline docs/library/graphing/brush.md:60]: is the brush for scaling the data axes?

**Review by masenf — CHANGES_REQUESTED** [review]
partial review, still going through

**Comment by Alek99** [comment]
This is not the final docs just keep that in mind I wanted to merge something in because I didn't want the pr to get too big

**Comment by Alek99** [comment]
I'll be adding follow ups to this

**Comment by tgberkeley** [comment]
Points to change:

can't add list … to funnel chart 

ordering of x axis on scatter chart

error bars taking in lists and integers, and x axis ones not rendering

what is reference line and how do we rename this component in the sidebar as here there are 3 components in one

change the XAxis title to x, y , z axes or something

**Comment by Alek99** [comment]
Ok addressing these comment rn will push a follow up tonight

**Comment by Alek99** [comment]
I've solved everything on that list except error bat made an issue on rechart because I believe it may be a bug https://github.com/recharts/recharts/issues/3811

**Review by tgberkeley — APPROVED** [review]

**Pull request merged**